app/app/my-markets/page.tsx

```
@@ -446,20 +463,24 @@ const MyMarketsPage: FC = () => {
     return () => { cancelled = true; };
   }, [myMarkets, connection]);
 
+  const pageHeader = (
+    <div className="mb-2 text-[10px] font-medium uppercase tracking-[0.25em] text-[var(--accent)]/60">
+      // admin
+    </div>
```

Biome (2.3.14)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)